src/devices/current_cost_envir.c

```diff
@@ -0,0 +1,107 @@
+/* CurrentCost EnviR Transmistter
+ * Contributed by Neil Cowburn <git@neilcowburn.com>
+ */
+
```

Comment:
Adde copyright header.

Reply:
All the requested changes, bar this one, should probably be applied to [current_cost.c](https://github.com/merbanan/rtl_433/blob/master/src/devices/current_cost.c) as well, since that's where I copied the code from. Would you like me to do that as part of this pull request?

Reply:
Open a new one so this is not delayed.